robotidy/transformers/SplitTooLongLine.py

```diff
@@ -0,0 +1,112 @@
+from itertools import tee
+
+from robot.api.parsing import (
+    ModelTransformer,
+    Token
+)
+
+
+def eol():
+    return Token(Token.EOL)
+
+
+class SplitTooLongLine(ModelTransformer):
+    def __init__(self, line_length: int = 120):
+        self.line_length = line_length
+
+    def visit_KeywordCall(self, node):  # noqa
+        if all(token.end_col_offset <= self.line_length for token in node.tokens[::-1]):
+            return node
+        if not node.get_tokens(Token.ARGUMENT):  # return if there are no arguments - nothing to split
```

Comment:
It will not work in case of one discussed example:
```
Keyword    # with bit longer comment that could be in different line
```

Reply:
Good catch. I didn't change much of `visit_KeywordCall` from your PoC yet btw., will do that as I go.